javascript/javascript1/week3/Javascript_warmup.js

```diff
@@ -0,0 +1,110 @@
+// #1 Item array removal
+
+const names = [
+    "Peter",
+    "Ahmad",
+    "Yana",
+    "kristina",
+    "Rasmus",
+    "Samuel",
+    "katrine",
+    "Tala",
+  ];
+  const nameToRemove = "Ahmaad";
+  
+  // Write some code here
+  
+  function removeName(name){
+    if(typeof(name) === "string" && name!= ""){
+
+      let indexOfName = names.indexOf(name);
+      if(indexOfName === -1){
+        console.log("Error! Incorrect name");
+        return 0; 
+      } 
+      else{
+        names.splice(indexOfName, 1);
+      }
+    } 
+    else{
+      return "Error! Incorrect name";
+    }
+    
+  }
+
+  removeName(nameToRemove);
+
+  // Code done
+  
+  console.log(names); // ['Peter', 'Yana', 'kristina', 'Rasmus', 'Samuel', 'katrine', 'Tala']
+
+
+
+  // #2 Speed function 
+
+  function calculateTime({speed, destinationDistance}){
+    x = typeof(speed) === "number" && typeof(destinationDistance) === "number";
```

Comment:
If you have complex boolean logic, it can be very good to break it up so it becomes more digestible and easy to read! 😄 It is paramount to pick good names for variables though! Can you find better names for `x` and `y` here? There is also an instance of this in the CactusIO implementation that I'd like you to revisit as well. 🙂

You are also missing `const` or `let`; which one would you choose here?